roles/lib_openshift/src/class/oc_env.py

```diff
@@ -0,0 +1,147 @@
+# pylint: skip-file
+# flake8: noqa
+
+
+# pylint: disable=too-many-instance-attributes
+class OCEnv(OpenShiftCLI):
+    ''' Class to wrap the oc command line tools '''
+
+    container_path = {"pod": "spec.containers[0].env",
+                      "dc":  "spec.template.spec.containers[0].env",
+                      "rc":  "spec.template.spec.containers[0].env",
+                     }
+
+    # pylint allows 5. we need 6
+    # pylint: disable=too-many-arguments
```

Comment:
Personally, I find limiting the number of arguments to be an arbitrary metric. I'm good with bumping it up to 10 (or even removing the check).

I'm wondering if we need to find some type of mechanism for tracking and voting on these types of issues?